Formula/hab.rb

```diff
@@ -15,7 +14,7 @@ class Hab < Formula
 
   desc "The Habitat command line application"
   homepage "https://habitat.sh"
-  url "https://bintray.com/habitat/stable/download_file?file_path=darwin%2Fx86_64%2Fhab-#{current_version}-#{current_release}-x86_64-darwin.zip"
+  url "https://packages.chef.io/files/stable/habitat/latest/hab-x86_64-darwin.zip"
```

Comment:
OK, so this is tricky... until 0.88.0 is in `packages.chef.io`, we are going to have to handle both locations until the next release is out (i.e., we should be able to merge this PR, have it still work for 0.88.0, _and also_ be ready to accept the changes for the next release.

We probably want to hard-code the complete current bintray URL (including version and release), and then do a test on `current_version`... if it's 0.88.0, use the bintray URL, otherwise, use packages.chef.io.